rellic/AST/Z3ConvVisitor.cpp

```diff
@@ -548,6 +556,14 @@ bool Z3ConvVisitor::VisitBinaryOperator(clang::BinaryOperator *c_op) {
       InsertZ3Expr(c_op, lhs != rhs);
       break;
 
+    case clang::BO_LAnd:
+      InsertZ3Expr(c_op, Z3BoolCast(lhs) && Z3BoolCast(rhs));
+      break;
+
+    case clang::BO_LOr:
+      InsertZ3Expr(c_op, Z3BoolCast(lhs) || Z3BoolCast(rhs));
+      break;
+
     case clang::BO_Rem:
       InsertZ3Expr(c_op, z3::srem(lhs, rhs));
```

Comment:
We should differentiate between signed and unsigned remainders here. If the `clang::BO_Rem` operation yields a signed result, we should use `z3::srem()`, `z3::urem()` otherwise.